cradle-cassandra/src/main/java/com/exactpro/cradle/cassandra/ReadThroughBookCache.java

```diff
@@ -0,0 +1,130 @@
+package com.exactpro.cradle.cassandra;
+
+import com.datastax.oss.driver.api.core.cql.BoundStatementBuilder;
+import com.exactpro.cradle.*;
+import com.exactpro.cradle.cassandra.dao.CradleOperators;
+import com.exactpro.cradle.cassandra.dao.books.BookEntity;
+import com.exactpro.cradle.cassandra.dao.books.PageEntity;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.function.Function;
+
+public class ReadThroughBookCache implements BookCache {
+    private static final Logger logger = LoggerFactory.getLogger(ReadThroughBookCache.class);
+
+    private final CradleOperators ops;
+    private final Map<BookId, BookInfo> books;
+    private final Function<BoundStatementBuilder, BoundStatementBuilder> readAttrs;
+    private final String schemaVersion;
+
+    public ReadThroughBookCache(CradleOperators ops,
+                                Function<BoundStatementBuilder, BoundStatementBuilder> readAttrs,
+                                String schemaVersion) {
+        this.ops = ops;
+        this.books = new ConcurrentHashMap<>();
+        this.readAttrs = readAttrs;
+        this.schemaVersion = schemaVersion;
+    }
+
+    public BookInfo getBook (BookId bookId) {
+        if (!books.containsKey(bookId)) {
+            logger.info("Could not find book named {} in cache, tyring getting from db", bookId.getName());
```

Comment:
Replace this line with the following:
`logger.info("Book '{}' is absent in cache, trying to get it from DB", bookId.getName());`